utils_nlp/models/transformers/common.py

```diff
@@ -212,16 +220,20 @@ def predict(self, eval_dataloader, get_inputs, num_gpus, gpu_ids, verbose=True):
                 logits = outputs[0]
             yield logits.detach().cpu().numpy()
 
-    def save_model(self):
-        output_model_dir = os.path.join(self.cache_dir, "fine_tuned")
-
-        os.makedirs(self.cache_dir, exist_ok=True)
-        os.makedirs(output_model_dir, exist_ok=True)
-
-        logger.info("Saving model checkpoint to %s", output_model_dir)
+    def save_model(self, full_name=None):
         # Save a trained model, configuration and tokenizer using `save_pretrained()`.
```

Comment:
Docstring needs to be updated. If full_name is provided, model is state_dict is saved and can be loaded by torch.load()

Reply:
will do